features/steps/gig_management.rb

```diff
@@ -0,0 +1,268 @@
+    end
+  end
+
+  step 'there is at least one published gig' do
+    @gig = FactoryGirl.create(:gig, published: true)
+  end
+
+  step 'there is at least one existing gig' do
+    @gig = FactoryGirl.create(:gig)
+  end
+
+  step 'I add a new gig' do
+    @new_gig = FactoryGirl.create(:gig)
+  end
+
+  step 'there is at least one unpublished gig' do
+    @unpublished_gig = FactoryGirl.create(:gig, published: false)
+  end
+
+  step 'I go to the Schedule Page' do
+    visit schedule_path
+  end
+
+  step 'I see the published gig' do
+    i_see_information_for_a_gig
+  end
+
+  step 'I go to the Schedule Page' do
+    visit schedule_path
+  end
+
+  step 'I do not see a Manage Gigs link' do
+    expect(page).to have_no_link("Manage Gigs")
+  end
+
+  step 'I do not see the unpublished gig' do
+    expect(find(".gig-id-#{@unpublished_gig.id}")).not_to have_content(@unpublished_gig.name)
+  end
+
+  step 'I do not see an Add Gig link' do
+    expect(page).to have_no_link("Add Gig")
+  end
+
+  step 'I do not see an Edit button' do
+    expect(page).not_to have_button("Edit")
+  end
+
+  step 'I do not see a Delete button' do
+    expect(page).not_to have_button("Delete")
+  end
+
+  step 'I navigate to the Sign In page' do
+    find("li.navigation").hover
+    i_click_sign_in
+    expect(page.title).to eq("Sign In")
+  end
+
+  step 'I see an alert containing "You must be signed in to access that page"' do
+    has_flash_msg(severity: :alert, containing: "You must be signed in to access that page")
+  end
+
+  step 'I see an alert containing "You must be an admin user to access that page"' do
+    has_flash_msg(severity: :alert, containing: "You must be an admin user to access that page")
+  end
+
+  step 'I see an error message' do
+    has_form_error
+  end
+
+  step 'I visit the Gig Management page directly' do
+    visit manage_gigs_path
+  end
+
+  step 'I visit the Add Gig page directly' do
+    visit new_gig_path
+  end
+
+  step 'I visit the Edit Gig page directly' do
+    visit edit_gig_path(1) # any user id will work for this test
+  end
+
+  step 'I change the published checkbox' do
+    if @gig.published?
+      @orig_published_value = true
+      uncheck "gig_published"
+    else
+      @orig_admin_value = false
+      check "gig_published"
+    end
+  end
+
+  step 'the published field is changed' do
+    within find(".gig-id-#{@gig.id}") do
+      if @orig_published_value
+        expect(page).not_to have_css("span.user_published")
+      else
+        expect(page).to have_css("span.user_published")
+      end
+    end
+  end
+
+  private
+
+  def verify_gig_schedule(gig)
+    id_class = ".gig-id-#{gig.id}"
+    expect(find(".schedule #{id_class} span.gig_name")).to have_content(gig.name)
+    expect(find(".schedule #{id_class} span.gig_note")).to have_content(gig.note)
+    expect(find(".schedule #{id_class} span.gig_location")).to have_content(gig.location)
+  end
+
+  def sync_title(text)
```

Comment:
I know.  It is not needed with very similar tests for the users.  But it IS needed here.   The reason needs to be investigated.